src/plugincollection.js

```diff
@@ -110,6 +166,29 @@ export default class PluginCollection {
 			} );
 		}
 
+		function getPluginConstructor( PluginConstructorOrName ) {
+			if ( typeof PluginConstructorOrName === 'function' ) {
+				return PluginConstructorOrName;
+			}
+
+			const PluginConstructor = that._availablePlugins.get( PluginConstructorOrName );
+
+			if ( !PluginConstructor ) {
+				/**
+				 * The loaded plugin module is not available.
```

Comment:
Update to:

```js
The plugin cannot be loaded by name.

Plugin classes need to be provided to the editor before they can be loaded by name. This is usually done by the builder.

TODO update this error with links to docs because it will be a very frequent problem.

@error plugincollection-plugin-not-found
@param {String} pluginName The name of the plugin which could not be loaded.
```